Libplanet/KeyStore/Web3KeyStore.cs

```diff
@@ -0,0 +1,185 @@
+        /// <exception cref="ArgumentNullException">Thrown when <c>null</c> is passed to
+        /// <paramref name="path"/>.</exception>
+        public Web3KeyStore(string path)
+        {
+            _logger = Log.ForContext<Web3KeyStore>().ForContext("DirectoryPath", path);
+
+            if (path is null)
+            {
+                throw new ArgumentNullException(nameof(path));
+            }
+            else if (!Directory.Exists(path))
+            {
+                Directory.CreateDirectory(path);
+                _logger.Debug(
+                    "Created a directory {DirectoryPath} as it did not exist.",
+                    path
+                );
+            }
+
+            Path = path;
+        }
+
+        /// <summary>
+        /// A default <see cref="Web3KeyStore"/> instance which refers to a user-local directory.
+        /// The <see cref="Path"/> differs on the platform:
+        /// <list type="table">
+        /// <listheader>
+        /// <term>OS</term>
+        /// <description>Directory path</description>
+        /// </listheader>
+        /// <item>
+        /// <term>Linux/macOS</term>
+        /// <description><var>$HOME</var>/.config/planetarium/keystore</description>
+        /// </item>
+        /// <item>
+        /// <term>Windows</term>
+        /// <description><var>%USERPROFILE%</var>\AppData\Roaming\planetarium\keystore</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public static Web3KeyStore DefaultStore =>
+            new Web3KeyStore(DefaultPath);
+
+        /// <summary>
+        /// The path of the directory key files are placed.
+        /// </summary>
+        public string Path { get; }
+
+        /// <inheritdoc/>
+        public IEnumerable<Tuple<Guid, ProtectedPrivateKey>> List() =>
+            ListFiles().Select(pair => Tuple.Create(pair.Item1, Get(pair.Item2)));
+
+        /// <inheritdoc/>
+        public IEnumerable<Guid> ListIds() =>
+            ListFiles().Select(pair => pair.Item1);
+
+        /// <inheritdoc/>
+        public ProtectedPrivateKey Get(Guid id)
+        {
+            string name;
+            try
+            {
+                (_, name) = ListFiles().First(pair => pair.Item1.Equals(id));
+            }
+            catch (InvalidOperationException)
+            {
+                throw new NoKeyException(id, "There is no key with such ID");
+            }
+
+            return Get(name);
+        }
+
+        /// <inheritdoc/>
+        public Guid Add(ProtectedPrivateKey key)
+        {
+            if (key is null)
+            {
+                throw new ArgumentNullException(nameof(key));
+            }
+
+            Guid keyId = Guid.NewGuid();
+            string filename = string.Format(
+                CultureInfo.InvariantCulture,
+                NameFormat,
+                DateTimeOffset.UtcNow,
+                keyId
+            );
+            var keyPath = System.IO.Path.Combine(Path, filename);
```

Comment:
How about using `Path` directly without namespace?

```suggestion
            var keyPath = Path.Combine(Path, filename);
```

Reply:
Without `System.IO.` qualification the compiler recognizes this to the property `Web3KeyStore.Path` instead of the class `System.IO.Path` due to ambiguity.